src/Microsoft.AspNetCore.Hosting/WebHostExtensions.cs

```diff
@@ -76,7 +77,7 @@ public static void Run(this IWebHost host)
             var done = new ManualResetEventSlim(false);
             using (var cts = new CancellationTokenSource())
             {
-                AttachCtrlcSigtermShutdown(cts, done, shutdownMessage: "Application is shutting down...");
+                AttachCtrlcSigtermShutdown(cts, done, shutdownMessage: host.Services.GetService<WebHostOptions>().WriteStatusMessages ? "Application is shutting down..." : string.Empty);
```

Comment:
GetRequiredService
break this line up / move the message definition to a local var.